src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -145,11 +145,26 @@ private String getJniPlatform( String os ) {
             default: throw new IllegalArgumentException("No support yet for " + os);
         }
     }
+    /*
+     * Make sure the clibaries needed for linking are available for this particular configuration.
+     * The clibraries path is available by default in GraalVM, but the directory for cross-platform libs may
+     * not exist. In that case, retrieve the libs from our download site.
+     */
+    void ensureClibs (ProjectConfiguration projectConfiguration) throws IOException {
+        Triplet target = projectConfiguration.getTargetTriplet();
+        Path clibPath = Path.of(projectConfiguration.getGraalPath(), "lib", "svm", "clibraries", target.getOsArch2());
+        if (!Files.exists(clibPath)) {
+            String url = "http://download2.gluonhq.com/substrate/clibs/"+target.getOsArch()+".zip";
```

Comment:
Move URL to static field, it can contain a variable that here will be replaced, like the `JAVA_STATIC_ZIP` case.

Reply:
better to make it a static field indeed.
However, it is very different from JAVA_STATIC_ZIP as it should never be modified or made customizable.

Reply:
It is customizable since you can set the `osArch`.

Reply:
I'd use something like `URL_CLIBS_ZIP = "http://download2.gluonhq.com/substrate/clibs/${osarch}.zip"`
and then it can be customized with: 
`String url = URL_CLIBS_ZIP.replace("${osarch}", target.getOsArch());`

Reply:
done